Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedGraphicsLayerSkia.cpp

```diff
@@ -91,6 +95,8 @@ Ref<Nicosia::Buffer> CoordinatedGraphicsLayer::paintTile(const IntRect& tileRect
 
     // Non-blocking, multi-threaded variant.
     if (auto* workerPool = m_coordinator->skiaUnacceleratedThreadedRenderingPool()) {
+        WTFBeginSignpost(this, PaintTile, "Skia unaccelerated multithread");
```

Comment:
This is useful, but even more useful would be to measure independently the time for the recording of the display list, and then in the postTask() lambda, the time for the replay + paint operation (ideally including the thread ID, so we can distinguish which thread was doing work -- or does sysprof already incldue that in the mark table, can't recall right now).

Reply:
@carlosgcampos 
I already suggested that and thought it was done.

Reply:
Done for real now. Sorry about this!